bedrock/mozorg/templates/mozorg/technology.html

```diff
@@ -29,6 +29,13 @@
 {% endblock %}
 
 {% block content %}
+
+{% if LANG.startswith('en-') %}
+  {% include 'mozorg/developer/includes/dev-tech-sub-nav.html' %}
+{% else %}
+  {% include 'mozorg/developer/includes/sub-nav.html' %}
```

Comment:
This page is localized, so a fallback like this probably isn't going to work (since this page does not contain the strings for the existing sub nav). I'd like to get @flodolo's input here, but an l10n tag is probably the easiest solution.

Reply:
That is very true, hmmmm. So what is the l10n tag you are referring to? I probably even know, just need a refresher.

Reply:
Sorry my bad, I should have explained a bit further. One similar to this https://github.com/mozilla/bedrock/blob/master/bedrock/firefox/templates/firefox/new/scene1.html#L103-L107

Reply:
So, that makes more sense but, would we not need the additional file anyhow? The reason I say that is because, the /developer page currently has a subnav while the technology page does not.

So, for the /developer page I can do a if l10n tag show the new subnav else the old but, for /technology there is no current subnav so, it should be if l10n tag include the new subnav else nothing.